index.html

```diff
@@ -727,15 +727,15 @@ <h2 id="securing-with-sd-jwt">Securing JSON-LD Verifiable Credentials with SD-JW
         <section>
             <h2 id="securing-vps-sd-jwt">Securing JSON-LD Verifiable Presentations with SD-JWT</h2>
             <p>
-                This section details how to use SD-JWT to secure verifiable presentations conforming
+                This section details how to use [[SD-JWT]] to secure verifiable presentations conforming
                 to [[VC-DATA-MODEL-2.0]].
             </p>
             <p>
                 A [=conforming SD-JWT issuer implementation=] MUST use [[SD-JWT]] to secure this media type.
-                The unsecured verifiable presentation is the unencoded SD-JWT payload.
+                The unsecured [=verifiable presentation=] is the unencoded [[SD-JWT]] payload.
             </p>
             <p>
-                The <code>typ</code> header parameter SHOULD be <code>vp-ld+sd-jwt</code>.
+                The <code>typ</code> header parameter SHOULD be <code>vp+sd-jwt</code>.
                 When present, the <code>cty</code> header parameter SHOULD be <code>vp</code>.
```

Comment:
```suggestion
                The <code>cty</code> header parameter MUST be <code>vp</code>.
```

Reply:
David, since this suggestion touches on language not proposed in this PR, please raise a separate issue to discuss your desire to strengthen the normative language here. The group agreed to use the 'cty' property, it was not decided how strongly to require it.

Reply:
Here is the wording of the resolution "We will note in particular that the cty property is the point of differentiation for others that may wish to use identical media types."
Thus is seems totally appropriate that this PR, which is meant to enact the resolution, should include text to show how that the differentiation is technically enforced.